### PR TITLE
Fix the README

### PR DESCRIPTION
**Pull request by Toby56**
I've tried to fix basically everything I can think of.

### CODE REVIEW

**Comment by Toby56** [comment]
For #38

File: README.md [hunk 1]
```diff
@@ -71,24 +74,24 @@ _NOTE_: To generate a complete `README.md`, be sure that the structure of your p
...
 
-This allows the tool to create a better README.md with cooler informations.
+This allows the tool to create a better `README.md` with more information.
 
 Here is a generated [README example](https://github.com/ibbatta/readme-generator/blob/master/EXAMPLE-README.md).
 
 ---
 
-## **:wrench: Developer usage**
+## **:wrench: Development**
```
Comment (ibbatta) [inline README.md]: These titles must not be changed

**Comment by Toby56** [comment]
It seems somebody else changed a some things as well....

**Comment by Toby56** [comment]
Just approve both of them and resolve the conflicts when you merge, right?

**Comment by Toby56** [comment]
Or maybe that's not the reason!
Well whatever do you want me to revert these four changes or are the more?

File: README.md [hunk 2]
```diff
@@ -8,19 +8,22 @@
...
 
-> This project was born mainly to satisfy my desire for knowledge and to simplify my workflow. Being a curious (and also lazy) developer I have always wondered how to avoid wasting my working time to write README.md files who fits perfectly project by project and that's why I wrote a tool to help me simplify and speed up these processes.
+> This project was born to simplify my workflow and satisfy my desire for knowledge. Being a curious (and also lazy) developer, I've always wandered how to avoid wasting time writing README.md files that fit perfectly project after project. That's why I wrote this tool to help simplify and speed up this process!
 
 ![Example gif](./.github/assets/example.gif)
 
 ---
 
-## **:clipboard: User usage**
+## Instalation
```
[the remark this answers is not in the document]
Comment (Toby56) [reply README.md]: Because of other users?

**Comment by ibbatta** [comment]
@Toby56 can you just revert the specified changes? (sorry for the delay I am a bit busy right now)

**Comment by Toby56** [comment]
I hope that OK sorry for the trouble

**Comment by Toby56** [comment]
Hey @ibbatta I hope that's OK, sorry for the trouble!

**Pull request merged**